src/io/io_response.cpp

```diff
@@ -14,35 +14,35 @@
 #include "../errors.hpp"
 #include "io_response.hpp"
 
-const std::string ResponseSink::STRINGS[] = {
-	"OK",       // ResponseCode::OK
-	"WHAT",     // ResponseCode::WHAT
-	"FAIL",     // ResponseCode::FAIL
-	"OHAI",     // ResponseCode::OHAI
-	"STATE",    // ResponseCode::STATE
-	"TIME",     // ResponseCode::TIME
-	"FILE",     // ResponseCode::FILE
-	"FEATURES", // ResponseCode::FEATURES
-	"END"       // ResponseCode::END
+const std::string Response::STRINGS[] = {
+	"OK",       // Code::OK
+	"WHAT",     // Code::WHAT
+	"FAIL",     // Code::FAIL
+	"OHAI",     // Code::OHAI
+	"STATE",    // Code::STATE
+	"TIME",     // Code::TIME
+	"FILE",     // Code::FILE
+	"FEATURES", // Code::FEATURES
+	"END"       // Code::END
 };
 
-void ResponseSink::Respond(ResponseCode code, const std::string &message) const
+Response::Response(Response::Code code)
 {
-	this->RespondArgs(code, std::vector<std::string>(1, message));
+	this->string = Response::STRINGS[static_cast<int>(code)];
 }
 
-void ResponseSink::RespondArgs(ResponseCode code,
-                               const std::vector<std::string> &arguments) const
+Response &Response::Arg(const std::string &arg)
```

Comment:
`+` or `<<`?  I'd be tempted to go for `+`, since concatenation is basically what's happening, whereas `<<` suggests some sort of stream operation (or bitshift, mwaha)